docs/sphinx/source/user_guide/index.rst

```diff
@@ -15,6 +15,6 @@ User Guide
    clearsky
    bifacial
    weather_data
-   variables_style_rules
+   glossary
```

Comment:
Can we move this line one below? I find super anti-intuitive the next topic is about SDE, then the FAQ:

```
   weather_data
   singlediode
   glossary
   faq
```